### PR TITLE
Merge "origin" and "origin_id" fields into "id"

### PR DESCRIPTION
**Pull request by spbnick**

### CODE REVIEW

File: kcidb/io/schema/__init__.py [hunk 1]
```diff
@@ -1,8 +1,11 @@
 """Kernel CI reporting I/O schema"""
 
 from kcidb.io.schema import v1
+from kcidb.io.schema import v2
```
Comment (gctucker) [inline kcidb/io/schema/__init__.py]: Do we need to keep a copy of each schema version? Are we actually planning to have backwards-compatibility support for this PoC?

Comment (spbnick) [reply kcidb/io/schema/__init__.py]: Yes. Apart from letting all submitters upgrade gradually, we will use this when upgrading the datasets to new schema (download, convert, upload).

Comment (gctucker) [reply kcidb/io/schema/__init__.py]: I see.  Is there an automated way of migrating a dataset from v1 to v2 from a db dump?

Comment (spbnick) [reply kcidb/io/schema/__init__.py]: Not completely automated, but there are instructions, in README.md.

**Comment by spbnick** [comment]
Thank you for the review, @gctucker! Merging!

**Pull request merged**